apps/condo/domains/resident/tasks.js

```diff
@@ -0,0 +1,72 @@
+const { getSchemaCtx } = require('@core/keystone/schema')
+const { createTask } = require('@core/keystone/tasks')
+
+const { sleep } = require('@condo/domains/common/utils/sleep')
+const { Property: PropertyAPI } = require('@condo/domains/property/utils/serverSchema')
+const { Resident: ResidentAPI } = require('@condo/domains/resident/utils/serverSchema')
+const { disconnectResidents, connectResidents } = require('@condo/domains/resident/utils/helpers')
+
+/**
+ * Reconnects residents to oldest (dominating) non-deleted property with same address
+ * on create, delete or restore operation on property with same address, or disconnects
+ * residents at all (connects to null) if there are no proper non-deleted property.
+ * @param propertyId
+ * @returns {Promise<void>}
+ */
+async function manageResidentToPropertyAndOrganizationConnections (propertyId, address, operationFlags = {}) {
```

Comment:
this logic should work correctly you don't pass the `operationFlags` argument. will it work like this?
Otherwise, it will be impossible to call this function somewhere in the CLI utility

Reply:
fixed